templates/auth0-singlelogout-handler.php

```diff
@@ -23,7 +23,10 @@
       domain:'<?php echo sanitize_text_field( $this->a0_options->get( 'domain' ) ); ?>'
     });
 
-    var sloOptions = { 'responseType' : 'token id_token', 'redirectUri' : '<?php echo home_url() ?>' };
```

Comment:
can this be considered a breaking change?

Reply:
No. `checkSession` just needs a valid callback, it doesn't actually send the browser there.